code/modules/food_and_drinks/pizzabox.dm

```diff
@@ -117,35 +117,38 @@
 		. += M
 
 /obj/item/pizzabox/attack_self(mob/user)
-	if(boxes.len > 0)
+	if(length(boxes) > 0)
 		return
 	open = !open
 	if(open && !bomb_defused)
 		audible_message(span_warning("[icon2html(src, hearers(src))] *beep*"))
 		bomb_active = TRUE
 		START_PROCESSING(SSobj, src)
-	else if(!open && !pizza && !bomb)
-		var/obj/item/stack/sheet/cardboard/cardboard = new /obj/item/stack/sheet/cardboard(user.drop_location())
-		to_chat(user, span_notice("You fold [src] into [cardboard]."))
-		user.put_in_active_hand(cardboard)
-		qdel(src)
-		return
 	update_appearance()
 
+/obj/item/pizzabox/attack_self_secondary(mob/user)
+	if(length(boxes) > 0)
+		return
+	if(pizza || bomb)
+		return
+	var/obj/item/stack/sheet/cardboard/cardboard = new /obj/item/stack/sheet/cardboard(user.drop_location())
+	balloon_alert(user, "deconstructed")
```

Comment:
```suggestion
```

This isn't necessary because there is already visual feedback where the user is looking (and the user is not where they are looking anyway)